### PR TITLE
Lesson learned from v0.6 cut

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
A couple lesson learned while cutting V0.6.0

### CODE REVIEW

File: bundle/manifests/nfd-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -527,4 +527,4 @@ spec:
   maturity: stable
   provider:
     name: kubernetes-sigs
-  version: 0.5.0
+  version: 0.6.0
```
Comment (marquiz) [inline bundle/manifests/nfd-operator.clusterserviceversion.yaml:530]: Should this be like `master` on the master branch?

File: bundle/manifests/nfd.kubernetes.io_nodefeaturediscoveries.yaml [hunk 2]
```diff
@@ -82,7 +90,7 @@ spec:
                 description: Deploy the NFD-Topology-Updater NFD-Topology-Updater
                   is a daemon responsible for examining allocated resources on a worker
                   node to account for resources available to be allocated to new pod
-                  on a per-zone basis https://kubernetes-sigs.github.io/node-feature-discovery/v0.10/get-started/introduction.html#nfd-topology-updater
+                  on a per-zone basis https://kubernetes-sigs.github.io/node-feature-discovery/v0.12/get-started/introduction.html#nfd-topology-updater
```
Comment (marquiz) [inline bundle/manifests/nfd.kubernetes.io_nodefeaturediscoveries.yaml:93]: Should we use `master` here, or `stable`? It kinda would make sense that the operand master branch try to be compatible with the master branch of the operand. Then on release branch we'd update these. WDYT?

File: config/manifests/bases/nfd-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -83,7 +83,7 @@ metadata:
     operators.operatorframework.io/builder: operator-sdk-v1.13.0+git
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/kubernetes-sigs/node-feature-discovery-operator
-  name: nfd-operator.v0.4.0
+  name: nfd-operator.v0.6.0
```
Comment (marquiz) [inline config/manifests/bases/nfd-operator.clusterserviceversion.yaml:86]: master, or v0.6.0?

File: config/manifests/bases/nfd-operator.clusterserviceversion.yaml [hunk 4]
```diff
@@ -571,11 +571,11 @@ spec:
   - discovery
   links:
   - name: Documentation operator
-    url: https://kubernetes-sigs.github.io/node-feature-discovery-operator/v0.4
+    url: https://kubernetes-sigs.github.io/node-feature-discovery-operator/v0.6
```
Comment (marquiz) [inline config/manifests/bases/nfd-operator.clusterserviceversion.yaml:574]: For the master branch should this be `master`. No need to update anymore then (only on release branches)

File: config/manifests/bases/nfd-operator.clusterserviceversion.yaml [hunk 5]
```diff
@@ -584,4 +584,4 @@ spec:
   maturity: stable
   provider:
     name: kubernetes-sigs
-  version: 0.4.0
+  version: 0.6.0
```
Comment (marquiz) [inline config/manifests/bases/nfd-operator.clusterserviceversion.yaml:587]: `master` as well(?)

File: docs/_config.yml [hunk 6]
```diff
@@ -42,15 +42,15 @@ remote_theme: rundocs/jekyll-rtd-theme@v2.0.10
 #   - vendor/ruby/
 
 # Document versioning
-version: master
+version: v0.6.0
```
Comment (marquiz) [inline docs/_config.yml:45]: this definitely needs to be master
```suggestion
version: master
```

File: docs/_config.yml [hunk 7]
```diff
@@ -42,15 +42,15 @@ remote_theme: rundocs/jekyll-rtd-theme@v2.0.10
...
 # in the sidebar (top left corner of the page)
-release: master
+release: v0.6.0
 
 # Container image which to point to in the documentation
-container_image: gcr.io/k8s-staging-nfd/node-feature-discovery-operator:master-minimal
+container_image: registry.k8s.io/nfd/node-feature-discovery-operator:v0.6.0-minimal
 # Operand Node Feature Discovery documentation version
-operand_version: master
+operand_version: v0.12
```
Comment (marquiz) [inline docs/_config.yml:56]: and this 
```suggestion
operand_version: master
```

File: config/manifests/bases/nfd-operator.clusterserviceversion.yaml (hunk 3 above)
Comment (ArangoGutierrez) [reply config/manifests/bases/nfd-operator.clusterserviceversion.yaml:86]: same, this is `operator-sdk` related, won't matter

File: bundle/manifests/node-feature-discovery-operator.clusterserviceversion.yaml [hunk 8]
```diff
@@ -14,7 +14,7 @@ metadata:
           "spec": {
             "instance": "",
             "operand": {
-              "image": "gcr.io/k8s-staging-nfd/node-feature-discovery:master",
+              "image": "registry.k8s.io/nfd/node-feature-discovery-operator:master-minimal",
```
Comment (marquiz) [inline bundle/manifests/node-feature-discovery-operator.clusterserviceversion.yaml:17]: must be staging repo, the official release repo does not have "master" images

File: bundle/manifests/node-feature-discovery-operator.clusterserviceversion.yaml [hunk 9]
```diff
@@ -403,8 +403,8 @@ spec:
                 - name: OPERATOR_NAME
                   value: cluster-nfd-operator
                 - name: NODE_FEATURE_DISCOVERY_IMAGE
-                  value: gcr.io/k8s-staging-nfd/node-feature-discovery:master
-                image: registry.k8s.io/nfd/node-feature-discovery-operator:0.5.0-minimal
+                  value: gcr.io/k8s-staging-nfd/node-feature-discovery:master-minimal
+                image: registry.k8s.io/nfd/node-feature-discovery-operator:master-minimal
```
Comment (marquiz) [inline bundle/manifests/node-feature-discovery-operator.clusterserviceversion.yaml:407]: staging

File: config/manager/kustomization.yaml [hunk 10]
```diff
@@ -13,4 +13,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: registry.k8s.io/nfd/node-feature-discovery-operator
-  newTag: 0.5.0
+  newTag: master
```
Comment (marquiz) [inline config/manager/kustomization.yaml:16]: you prolly need to set the registry above to staging(?)

File: deploy/helm/nfd-operator/crds/nfd-api-crds.yaml [hunk 11]
```diff
@@ -90,7 +90,7 @@ spec:
                 description: Deploy the NFD-Topology-Updater NFD-Topology-Updater
                   is a daemon responsible for examining allocated resources on a worker
                   node to account for resources available to be allocated to new pod
-                  on a per-zone basis https://kubernetes-sigs.github.io/node-feature-discovery/v0.10/get-started/introduction.html#nfd-topology-updater
+                  on a per-zone basis https://kubernetes-sigs.github.io/node-feature-discovery/v0.12/get-started/introduction.html#nfd-topology-updater
```
Comment (marquiz) [inline deploy/helm/nfd-operator/crds/nfd-api-crds.yaml:93]: master(?)

File: bundle/manifests/node-feature-discovery-operator.clusterserviceversion.yaml (hunk 9 above)
Comment (marquiz) [inline bundle/manifests/node-feature-discovery-operator.clusterserviceversion.yaml:407]: staging

Comment (ArangoGutierrez) [reply bundle/manifests/node-feature-discovery-operator.clusterserviceversion.yaml]: done!

File: .github/ISSUE_TEMPLATE/release.md [hunk 12]
```diff
@@ -18,6 +18,7 @@ Please do not remove items from the checklist
   - [ ] An OWNER pushes the newly created branch
         `git push release-0.$MAJ`
 - [ ] Update references in `docs/_config.yml` (`version`, `release` and `container_image` fields) to point to the upcoming release and subbmit a PR against the release branch
+- [ ] Update the
```
[the remark this answers is not in the document]
Comment (ArangoGutierrez) [reply .github/ISSUE_TEMPLATE/release.md:21]: OMG why vscode didn't added the full line plop

**Pull request merged**